### PR TITLE
Adding an art page

### PR DESCRIPTION
**Pull request by EgbertRijke**
Screenshot 2023-09-15 at 00 48 46

---------

Co-authored-by: Andrej Bauer <andrej.bauer@andrej.com>
Co-authored-by: Matej Petković <Petkomat@users.noreply.github.com>

### CODE REVIEW

**Comment by EgbertRijke** [comment]
I see that the caption didin't display. Let me tweak it a little bit.

File: ART.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Agda-unimath art
+
+<div align="center">
+  <img src="agda-unimath-black-and-gold.png" alt="Concepts in the agda-unimath library. Bauer and Petkovic, 2023" width="80%">
```
Comment (VojtechStep) [inline ART.md:4]: Maybe copy the alt to the body of the page as well, so that other people know what the art represents?

Comment (EgbertRijke) [reply ART.md]: Did that. Thanks for noticing!

**Comment by EgbertRijke** [comment]
It should be done now

**Review by fredrik-bakke — APPROVED** [review]
Looks good to me, although I haven't tried it locally

**Comment by EgbertRijke** [comment]
The updated result is displayed in the picture

File: theme/js/custom.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ if (link) {
   // console.log(link);
   var filename = link.getAttribute('href');
   const fileList = [
```
Comment (fredrik-bakke) [inline theme/js/custom.js:5]: Hmm, should `MIXFIX-OPERATORS.md` be in this list? 😬

Comment (EgbertRijke) [reply theme/js/custom.js:5]: Yup

Comment (EgbertRijke) [reply theme/js/custom.js:5]: Let me sneak it in real quick

Comment (fredrik-bakke) [reply theme/js/custom.js:5]: Sorry about that 😅

File: ART.md (hunk 1 above)
Comment (VojtechStep) [reply ART.md]: For future reference, the `alt` attribute isn't semantically the conventional "caption", i.e. a description of the thing which you always want to show next to the thing, but rather an "accessibility" feature, which replaces the image when visuals aren't available (e.g. the image can't load or the user is using a screen reader).

**Comment by fredrik-bakke** [comment]
Alright, this can be merged when you feel like it!

**Comment by EgbertRijke** [comment]
I tested it again and it works

**Pull request merged**